File: Modul-I/C#PartTwo/Homework/Arrays/PrimeNumbers/PrimeNumbers.cs
namespace PrimeNumbers
{
    using System;

    class PrimeNumbers
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());

            for (int j = n; j >= 0; j--)
            {
                int number = j;
                bool isPrime = true;

                for (int i = 2; i <= (int)Math.Sqrt(number); i++)
                {
                    if (number % i == 0)
                    {
                        isPrime = false;
                        break;
                    }
                }

                if (isPrime)
                {
                    Console.WriteLine(number);
                    break;
                }
            }


        }
    }
}

File: Modul-I/JavaScript-Fundamentals/Homeworks/07. Arrays/10. Prime numbers/PrimeNumbers.js
function solve(args){
    var n = +args[0],
        j,
        i,
        isPrime,
        number;

    for ( j = n; j >= 0; j-=1)
    {
         number = j;
         isPrime = true;

        for ( i = 2; i <= Math.sqrt(number); i+=1)
        {
            if (number % i == 0)
            {
                isPrime = false;
                break;
            }
        }

        if (isPrime)
        {
            console.log(number);
            break;
        }
    }

}